dashboard/db/migrate/20170131165950_add_uniqueness_to_user_proficiency_user_id.rb

```diff
@@ -0,0 +1,11 @@
+class AddUniquenessToUserProficiencyUserId < ActiveRecord::Migration[5.0]
+  def up
```

Comment:
can't you just do basically the same thing as the `up`, but with `unique: false` passed to `add_index`?